package.json

```diff
@@ -2864,6 +2864,150 @@
 												]
 											}
 										]
+									},
+									{
+										"key": "languageId:python",
+										"name": "Python",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "d",
+														"name": "+Format document with",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "m",
+														"name": "Format modified lines only",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "S",
+														"name": "+Format selection with",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "c",
+												"name": "+Execute",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "c",
+														"name": "Execute file in terminal",
+														"type": "command",
+														"command": "python.execInTerminal"
+													},
+													{
+														"key": "C",
+														"name": "Execute file in terminal",
+														"type": "command",
+														"command": "python.execInTerminal"
+													}
```

Comment:
why you duplicated if it's the same of `spc m c`?

Reply:
I wanted to replicate as closely as possible the Python layer.

Both `c` and `C` [exist](https://develop.spacemacs.org/layers/+lang/python/README.html#running-python-script-in-shell) . Their behaviour is nearly the same except that I can't map the exact difference in VSCode. I preferred to map it anyway so that people who have muscle-memory of this chord are not surprised if they enter it.

Reply:
sorry, missed that, great!